test/e2e/map-search.test.js

```diff
@@ -52,4 +52,25 @@ describe('vl-map-search', async () => {
     await assert.eventually.equal(search.getSelectedValue(), 'Temse');
     await assert.eventually.isTrue(map.hasZoom(5));
   });
+
+  it('als gebruiker kan ik zoeken op Lambert coördinaat en zal de kaart zoomen', async () => {
+    const map = await vlMapPage.getMap();
+    const search = await map.getSearch();
+
+    await search.open();
+    await search.zoomTo('104719.27, 192387.25');
+
+    await assert.eventually.isTrue(map.hasZoom(10));
```

Comment:
misschien ook valideren dat de kaart op de juiste plaats gezoomed is?

Reply:
ja center opvragen of zo en zien dat dat "dicht genoeg in de buurt" ligt

Reply:
moet eens bekijken hoe ik dat juist voor mekaar krijg

Reply:
Tot op heden gaan we niet zo ver op E2E niveau, lijkt me eerder iets om te valideren op unit niveau.

Reply:
Nu weet je enkel dat je zoomniveau is aangepast, niet dat je ook je view hebt verplaatst (wat ook een verwacht resultaat is, naast het aanpassen van het zoomniveau). 
Op unit test niveau test je enkel dat je de juiste functie aanroept op de vl-mapactions, wat niet hetzelfde is als testen dat het effectief iets doet op je scherm.